devsetup/Makefile

```diff
@@ -37,3 +38,26 @@ download_tools: ## Runs an ansible playbook to install required tools with the v
 	ANSIBLE_FORCE_COLOR=true ansible-playbook \
 	-v -i hosts \
 	download_tools.yaml
+
+.PHONY: crc_attach_default_interface
+crc_attach_default_interface:
+	sudo virsh attach-interface crc --source default --type network --model virtio --config --persistent
+
+.PHONY: edpm_compute
+edpm_compute:
```

Comment:
so this is really jsut creating the vms which is fine for testing with CRC

we might want to explore using CNV for this for CI/dev in the future instead.
but this is fine for now